test/e2e/helpers.js

```diff
@@ -655,7 +655,7 @@ const DAPP_URL = 'http://127.0.0.1:8080';
 const DAPP_ONE_URL = 'http://127.0.0.1:8081';
 
 const openDapp = async (driver, contract = null, dappURL = DAPP_URL) => {
```

Comment:
This is needed for the update to test/e2e/json-rpc/switchEthereumChain.spec.js, which can be seen below